### ISSUE TITLE
Illustration of the Dragon Soul and Power Word: Shield / Penance

### ISSUE THREAD

**Issue by StheNoble** [dalaranwow/dalaran-wow#4687]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: 
- Equip the Trinket "Illustration of the Dragon Soul" (Item ID 40432) on a Discipline Priest.
- Use Power Word: Shield on any target
- Notice that you don't gain any Stack of "Illustration of the Dragon Soul" or Refresh any allready existing Stacks.

**Expected behaviour**: 
- The cast of Power Word: Shield should give a "Illustration of the Dragon Soul" Stack or refresh existing ones.

**Steps to reproduce the problem**:
see current behaviour

**Include proofs for this behaviour**
Follwoing is a list of comments on WoWHead regarding this Trinket in conjuction with Power Word: Shield:
- http://www.wowhead.com/item=40432/illustration-of-the-dragon-soul#comments:id=484105
- http://www.wowhead.com/item=40432/illustration-of-the-dragon-soul#comments:id=738178
- http://www.wowhead.com/item=40432/illustration-of-the-dragon-soul#comments:id=638539

i also searched Worldoflogs for some Retail Combatlogs to support this Bugreport. The Following is an excerpt of a Combatlog filtered to a Disc Priest Power Word: Shield casts and his Dragon Soul Buff:
```
[20:56:26.161] Beatific casts Power Word: Shield on Sven
[20:56:26.161] Beatific's Illustration of the Dragon Soul is refreshed by Beatific
[20:56:26.505] Sven gains Power Word: Shield from Beatific
[20:56:27.685] Beatific casts Power Word: Shield on Velrin
[20:56:27.692] Beatific's Illustration of the Dragon Soul is refreshed by Beatific
[20:56:27.737] Velrin gains Power Word: Shield from Beatific
[20:56:30.498] Beatific's Illustration of the Dragon Soul is refreshed by Beatific
[20:56:40.565] Beatific's Illustration of the Dragon Soul fades from Beatific
[20:56:42.203] Beatific casts Power Word: Shield on Gronthor
[20:56:42.203] Beatific gains Illustration of the Dragon Soul from Beatific
[20:56:42.213] Gronthor gains Power Word: Shield from Beatific
```
Source: http://www.worldoflogs.com/reports/AQf9iUWnuHUdRCOH/log/

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Illustration of the Dragon Soul - http://wotlk.openwow.com/item=40432

Power Word: Shield (Rank 1) - http://wotlk.openwow.com/spell=17
up to
Power Word: Shield (Rank 14) - http://wotlk.openwow.com/spell=48066

**Include Screenshots from the issue if necessary**
/

**Comment by TeridanDW** [comment]
Hello, thank you for the report @StheNoble !
I can confirm the issue and also add that this is working on TC (tested on my server with spell http://wotlk.openwow.com/spell=17 rank 1 to 14, all of them proced the trinket's effect).

I would like to add to this report that spell http://wotlk.openwow.com/spell=47540 rank 1 to 4 should also proc the trinket's effect, but currently it doesn't.
Proof:

> By mits247 (158 – 1) on 2008/12/02 (Patch 3.0.3)		
> I can confirm now that priest's Penance now procs 3, thats right, 3 of these stacks in one channeling spell.

from http://www.wowhead.com/item=40432/illustration-of-the-dragon-soul#comments:id=484105

**Comment by Rushor** [comment]
-- Duplicate Issue --
Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/1745 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

Fix proof: http://i.imgur.com/Ci5pZzi.jpg
